### PR TITLE
Update environment-variables.md

### PR DESCRIPTION
**Pull request by abalmukund**
Updated command line details for Windows based systems. In Windows OS `echo $SAUCE_USERNAME` doesn't work.

### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Documentation fix (typos, incorrect content, missing content, etc.)

### Checklist

- [x] I have read the [contributing](https://github.com/saucelabs/sauce-docs/blob/main/docs/contributing.md) document.
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by abalmukund** [comment]
Only changed the documentation. Build and test is not required.

File: docs/basics/environment-variables.md [hunk 1]
```diff
@@ -43,7 +43,7 @@ For a list of Sauce Connect Proxy environment variables, see [Environment Variab
 7. For **Variable value**, enter your Sauce username.
 8. Click **OK**.
 9. Repeat 4 - 8 to set up the **SAUCE_ACCESS_KEY**.
-10. Confirm that your environment variables have been set by typing `echo $SAUCE_USERNAME` in your terminal. The response should be your username value.  Then do the same for your access key. 
+10. Confirm that your environment variables have been set by typing `echo %SAUCE_USERNAME%` in your terminal. The response should be your username value.  Then do the same for your access key.
```
Comment (sweeneyskirt-sl) [inline docs/basics/environment-variables.md:46]: Thanks for this correction, @abalmukund. Is the new value universal? `echo $SAUCE_USERNAME` works for me in BASH and ZSH. Why are you thinking it doesn't work? Should we instead provide a note saying that the verification command may be different if you are using a shell other than BASH or ZSH? Should we provide a few of the most common shell commands for this action? Let's think this through a bit before we just change the command, which, incidentally, is given in more places than just this file. @kimsauce @spider-sauce

**Review by sweeneyskirt-sl — APPROVED** [review]

**Comment by kimsauce** [comment]
Hi @sweeneyskirt-sl , the merge was failing due to what looks like an Algolia error. I've applied the change in a separate PR and will close this one. Thank you, @abalmukund for the catch!

**Comment by sweeneyskirt-sl** [comment]
Thanks, @kimsauce . I am not sure why this is still happening. I thought we put the algolia key back in our docusaurus config to mitigate this issue. @spider-sauce  -- not at all urgent, but maybe we can just get this in the backlog as something to look at?

**Comment by spider-sauce** [comment]
@kimsauce @sweeneyskirt-sl , using the `%` is standard for Windows env variables. `$` is for unix-based systems. The algolia key issue shouldn't be happening anymore. The public-safe key is hardcoded into the config. Ping me privately if you're still having this problem.

**Comment by sweeneyskirt-sl** [comment]
Thanks, @spider-sauce! I realized that the correction was specific to windows after I expanded the DIFF file in the replacement PR that Kim made, so the edit has been approved and merged. Regarding the algolia failings, it seems it does still happen for PRs submitted by contributors who create a fork of our repo and then submit a PR from their fork. It does not happen for contributors who just clone the repo and submit a PR from a branch to main.

**Pull request closed without merging**